src/actions/asset.js

```diff
@@ -97,15 +93,19 @@ export default class Asset {
         switch (action) {
             case 'ADD':
                 dispatchObject[this.assetType] = {
-                    ...asset.asset.data[this.assetType],
+                    assetData: asset.asset.data[this.assetType],
```

Comment:
you can use the spread operators here, the frontend only cares about the fields in `assetData` and `metaData`, not where they are stored in the transaction